test/v2/fixtures/retrieveBalanceAndTransactions/CodingDetailsFixture.scala

```diff
@@ -43,9 +46,9 @@ object CodingDetailsFixture {
   val codingDetailsMtdResponseJson: JsValue = Json.parse(
     s"""
        |{
-       |   "taxYearReturn": "${codingDetails.taxYearReturn.get}",
+       |   "returnTaxYear": "${codingDetails.returnTaxYear.get}",
        |   "totalLiabilityAmount": ${codingDetails.totalLiabilityAmount.get},
-       |   "taxYearCoding": "${codingDetails.taxYearCoding.get}",
+       |   "codingTaxYear": "${codingDetails.codingTaxYear.get}",
        |   "coded": {
        |      "charge": ${coded.charge.get},
        |      "initiationDate": "${coded.initiationDate.get}"
```

Comment:
I wonder if it is a good idea to use ".get" for all the variables in this file. Should we consider replacing this with ".getOrElse" or even using a string directly? https://stackoverflow.com/questions/25016245/what-does-the-get-method-do-in-scala

Reply:
I have hardcoded the JSONs as per our conversation.Thanks.